### DISCUSSION TITLE
Invalid or unsupported stamp [sdns://AgUAAAAAAAAAFSBbMmEwMjo2Yjg6OmZlZWQ6MGZmXQAZY29tbW9uLmRvdC5kbnMueWFuZGV4Lm5ldAovZG5zLXF1ZXJ5]: Invalid stamp (IP address)

### DISCUSSION THREAD

**Discussion by reliant8307** [DNSCrypt/dnscrypt-proxy#2570] [Q&A] [answered]
A few days ago, the log started to show the following error. Nothing was changed here. Are you seeing this same message on your logs?

`
Jan 21 18:44:24 tux dnscrypt-proxy[23623]: Error in source [public-resolvers]: [Invalid or unsupported stamp [sdns://AgUAAAAAAAAAFSBbMmEwMjo2Yjg6OmZlZWQ6MGZmXQAZY29tbW9uLmRvdC5kbnMueWFuZGV4Lm5ldAovZG5zLXF1ZXJ5]: Invalid stamp (IP address), Invalid or unsupported stamp [sdns://AgUAAAAAAAAAFSBbMmEwMjo2Yjg6OmZlZWQ6YTExXQAZZmFtaWx5LmRvdC5kbnMueWFuZGV4Lm5ldAovZG5zLXF1ZXJ5]: Invalid stamp (IP address), Invalid or unsupported stamp [sdns://AgEAAAAAAAAAFSBbMmEwMjo2Yjg6OmZlZWQ6YmFkXQAXc2FmZS5kb3QuZG5zLnlhbmRleC5uZXQKL2Rucy1xdWVyeQ]: Invalid stamp (IP address)] -- Continuing with reduced server count [388]
`

Any hints? Thanks!

**Comment by jedisct1** [comment, marked as answer]
Yes, that has been fixed since. You can delete the `public-resolvers.md` file to force an update, or simply wait for the next automatic update of the resolvers list.
